src/module/Subject/src/Subject/Manager/SubjectManager.php

```diff
@@ -95,10 +95,10 @@ protected function isTrashed(EntityInterface $entity, Collection $collection)
     public function getUnrevisedRevisions(TaxonomyTermInterface $term)
     {
         $entities = $this->entityManager->findAllUnrevised();
```

Comment:
`$entities` contains all the unrevisesd entities, however this includes children, which don't know the taxonomy...